packages/portfolio-contract/src/type-guards.ts

```diff
@@ -221,6 +221,12 @@ type FlowStatus = {
   error?: string;
 };
 
+export type EvmAccountStatus = {
+  nonce: number;
+  lca: null | `${string}1${string}`;
+  remoteAddress: null | `0x${string}`;
+};
```

Comment:
Consider adding status since we are calling XYZStatus:
```suggestion
export type EvmAccountStatus = {
    nonce: bigint;
    status: 'pending' | 'creating' | 'created' | 'failed';
    lca: null | `${string}1${string}`;
    remoteAddress: null | `0x${string}`;
    error?: string;
  }```

Reply:
please don't add `EvmAccountStatus`

see [earlier comment](https://github.com/Agoric/agoric-sdk/pull/11662#discussion_r2231392356)